Xamarin.Forms.Core/AutofitTextMode.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	[TypeConverter(typeof(AutoFitTextModeConverter))]
+	public enum AutoFitTextMode
+	{
+		None,
+		FitToWidth
+	}
+
+	[Xaml.TypeConversion(typeof(AutoFitTextMode))]
+	public class AutoFitTextModeConverter : TypeConverter
```

Comment:
I don't think we need a converter. both xaml inflaters know how to resolve enum values

Reply:
removed! ✂